mowgli_etl/pipeline/wdc/data_exploration/buckets.csv

```diff
@@ -1,93 +1,37 @@
-,,,,
-product name epson stylus photo inkjet x photo printer,,cambro shelving unit plastic camshelves,john boos tuscan island culinary depot,
-,,,,
-epson stylus photo inkjet x printer specs printers,,,,
-,,,,
-measures h x l includes hellip,,,,
-,,,,
-you flowers com,,,,
-,,,,
-efficiency,,,,
-,,,,
-frequency mhz gain dbi dbi gain dbd dbd,,,,
-,,,,
-signal,,,,
-,,,,
-Accessories,,,,
-,,,,
-der inline controller bietet die m glichkeit ber profinet und modbus tcp zu kommunizieren die programmierung erfolgt mit pc worx express oder pc worx iec,,,,
-,,,,
-typ npn pnp benennung ausgang digitale ausg nge anzahl der ausg nge anschlusstechnik leiter maximaler ausgangsstrom je kanal ma ohne analogen eingang ja ohne analogen ausgang ja ohne puls richtungs ausgang ja ohne z hlereingang ja programmierwerkzeug pc worx pc worx express programmspeicher kbyte k anweisungen il datenspeicher kbyte remanenter datenspeicher kbyte nvram anzahl steuerungs tasks echtzeituhr ja,,,,
-,,,,
-control,,,,
-,,,,
-buy bradburnhome moonshine petal ceramic votive h x w x d,,,,
-,,,,
-bradburnhome moonshine petal ceramic votive h x w d staples,,,,
-,,,,
-nos produits,,,,
-,,,,
-r f rence marque maxflags pavoisement et communication publicitaire dimensions env x cm mati res mat riels tissu polyester,,,,
-,,,,
-monsieur,,,,
-,,,,
-furniture,,,,
-,,,,
-item details product i,,,,
-,,,,
-arm chair,,,,
-,,,,
-mixology tooth pick stirstick organizer h stainless steel,,,,
-,,,,
-manufacturer cal mil plastics model number cal packed,,,,
-,,,,
-cal mil plastics condiment caddy countertop organizer,,,,
-,,,,
-Clothing,,,,
-,,,,
-garment dimensions size chest body length sleeve s m l xl,,,,
-,,,,
-hanes mens tagless t shirt,,,,
-,,,,
-x cm haken zum aufh ngen rv f cher integrierter spiegel viele steckunterteilungen zahnb rstenhalter hauptstoff polyester d polyurethane coated kontrastmaterial polyamid d baby ripstop polyurethane coated futter polyamid d polyurethane coated,,,,
-,,,,
-kulturbeutel tecowrap marine vaude fairer zum wickeln grundstoff net,,,,
-,,,,
-men s sizes,,,,
-,,,,
-waist hip xs sm md l xl shoes girls us euro uk inches centimeters shoes guys us euro uk inches centimeters,,,,
-,,,,
-topic,,,,
-,,,,
-Beauty,,,,
-,,,,
-dise ador ronan y erwan bouroullec fabricante axor colecci,,,,
-,,,,
-axor bouroullec bater,,,,
-,,,,
-hand carved detail,,,,
-,,,,
-manor queen bed,,,,
-,,,,
-size chart size size range bust size,,,,
-,,,,
-ladies decatur pima,,,,
-,,,,
-storage shelfblue h x folded w x folded d,,,,
-,,,,
-bookcases quill com,,,,
-,,,,
-w vdc input range,,,,
-,,,,
-output chassis mount vdc isolation dc converter models cui,,,,
-,,,,
-insert size w x h cord length weight lbs,,,,
-,,,,
-quote,,,,
-,,,,
-acrylic sign holder,,,,
+Electronics (volume: 154.08),,camshelving elements starter unit w x l x h withstands temperatures (volume: 55296.0),tuscan island l x w x h (volume: 67584.0),
```

Comment:
Looks like there's just a few items larger than bucket 1, which I'd guess means the bucket ranges are mis-calibrated.